src/librustc_resolve/lib.rs

```diff
@@ -65,16 +65,16 @@ use std::mem::replace;
 use rustc_data_structures::ptr_key::PtrKey;
 use rustc_data_structures::sync::Lrc;
 
-use error_reporting::{find_span_of_binding_until_next_binding, extend_span_to_previous_binding};
+use diagnostics::{find_span_of_binding_until_next_binding, extend_span_to_previous_binding};
 use resolve_imports::{ImportDirective, ImportDirectiveSubclass, NameResolution, ImportResolver};
 use macros::{InvocationData, LegacyBinding, ParentScope};
 
 type Def = def::Def<NodeId>;
 
 // N.B., this module needs to be declared first so diagnostics are
```

Comment:
Should I replace this `diagnostics` with `error codes`?

Reply:
Not, this is the "good" diagnostics.rs.

Reply:
Okay.